### ISSUE TITLE
Window-list & GWL not highlighting windows demanding attention.

### ISSUE THREAD

**Issue by smurphos** [linuxmint/cinnamon#10750]
```
 * Cinnamon version 5.2.7
   - Please specify if you are using the daily builds PPA
     (https://launchpad.net/~linuxmint-daily-build-team/+archive/ubuntu/daily-builds).
Nope
 * Distribution - Mint 20.3
 * Graphics hardware *and* driver used
 * steve@steve-Inspiron-5580:~$ inxi -G
Graphics:  Device-1: Intel UHD Graphics 620 driver: i915 v: kernel
           Device-2: Realtek Integrated_Webcam_HD type: USB driver: uvcvideo
           Display: x11 server: X.Org 1.20.13 driver: loaded: modesetting unloaded: fbdev,vesa
           resolution: 1920x1080~60Hz
           OpenGL: renderer: Mesa Intel UHD Graphics 620 (WHL GT2) v: 4.6 Mesa 21.2.6
 * 64 bit
 * Attach ~/.xsession-errors, or /var/log/syslog
 ```

**Issue**

I'm not sure that either the window list or the grouped-window-list are highlighting windows that demand attention correctly

**Steps to reproduce**
Open Nemo, minimise nemo, open a terminal - run `xprop -id $(wmctrl -lx | grep nemo.Nemo | cut -d" " -f1) -f _NET_WM_STATE 32a -set _NET_WM_STATE "_NET_WM_STATE_DEMANDS_ATTENTION"`

Verify it's been set if needed with `xwininfo -id $(wmctrl -lx | grep nemo.Nemo | cut -d" " -f1) -wm`

**Expected behaviour**

I'd expect this to result in the window-list / grouped window list indicating that the nemo window requires attention. This isn't happening (checked with various themes, just in case). 

**Other information**

**Configuration info, if applicable**
If this bug report is related to an Applet, Desklet or Extension, please paste (or use a pastebin service) the offending extension's settings, if it has any.  You can obtain this by opening its configuration, clicking the 'hamburger' button in the upper-right corner of the window, and selecting "Export to a file". **Please be sure to review the contents and remove any personal data it may contain.**

```

Please paste here

```

**Comment by smurphos** [comment]
Urgent signals appear to be working OK - I get them on the GWL and Window List from Virtualbox when a VM screenlocks. I'm assuming they are urgent signals as the window doesn't have the Demands Attention state set at the time.

**Comment by dexx086** [comment]
I run into an problem which could be related to this issue.

When I started to use GWL instead of simple Window List, I noticed that applications that requires attention are \_\_not blinking\_\_, while with the Window List I had this visual. For eg.: when I receive a message on Teams, with GWL nothing happens, while with Window List the application's icon is blinking until I focus Team's window.

In both cases the notification applet is working as expected (I see the unread message in the notification list, until I open teams), but it's not great that I have no visualization in GWL when an event occurs in an application.

Is there any reason why it works in Window List, but not in GWL?
Is this related to @smurphos's original issue? I tested his example, but it doesn't indicate blinking visualization neither on Window List (which at least works for the "basic" attention required status), so it might be a different issue. However, it indicates to me that this 'demands attention' status is only partially handled in Window List, and not at all in GWL.

Any hope to make GWL to properly visualize at least the "basic" attention requirement for an application?

**Comment by rcalixte** [comment]
> Any hope to make GWL to properly visualize at least the "basic" attention requirement for an application?

Until this can be worked on, have you tried any of the alternative window list applets available? Some of them are more actively developed and more importantly, they allow for faster development since they can be updated independent of Cinnamon.

**Comment by dexx086** [comment]
@rcalixte Yes, I tried all of them which are available in Cinnamon (like Cassia, Cobi), but none of them seemed to visualize anything when an application's window requires attention. So far, I saw any visuals only with the simple "Window list" applet.
I did just a quick install/try/uninstall for the other applets, if somebody says they should work, then I'll try to test more (maybe with a different theme, with more apps than just Teams, etc...), but so far I didn't see any promising behaviour with any of them.

**Comment by fejjj** [comment]
This is still an issue
